exonum-java-binding-core/src/test/java/com/exonum/binding/service/adapters/UserServiceAdapterTest.java

```diff
@@ -223,4 +224,53 @@ public void mountPublicApiHandler_FailsOnSubsequentCalls() {
     expectedException.expect(IllegalStateException.class);
     serviceAdapter.mountPublicApiHandler(0x0B);
   }
+
+  @Test
+  public void afterCommit_ValidatorNode() {
+    Snapshot snapshot = mock(Snapshot.class);
```

Comment:
Just for your taste: how about to refactor it to class field `@Mock  Snapshot snapshot`?